.travis.yml

```diff
@@ -5,6 +5,7 @@ sudo: false
 cache:
   directories:
     - $HOME/.composer/cache/files
+    - vendor
```

Comment:
no need to cache the vendor folder, as the composer cache is already persisted (and will be used when switching to ``prefer-dist``.
And avoiding the cache on the vendor folder avoids invaliding the cache on each build (due to the randomness in the autoloader file) and also avoids weird issues in case the cache gets corrupt (a corruption in the composer cache will force a redownload in composer, while a corruption in the vendor folder will keep using the package with a broken content)